libs/gltfio/src/FFilamentAsset.h

```diff
@@ -210,6 +218,7 @@ struct FFilamentAsset : public FilamentAsset {
     utils::NameComponentManager* mNameManager;
     std::vector<uint8_t> mGlbData;
     std::vector<utils::Entity> mEntities;
+    std::vector<utils::Entity> mLightEntities;
```

Comment:
These need to be destroyed in the FFilamentAsset destructor.

Reply:
`mLightEntities` is a subset of `mEntities`, and all of `mEntities` get destroyed. If I'm not mistaken, this should be sufficient to destroy their Light components:

```
    void destroy(utils::Entity e);              //!< Destroys all filament-known components from this entity

```

Reply:
Doh!  Cool.